pkg/helm/client.go

```diff
@@ -474,14 +474,16 @@ func (h *Client) test(ctx context.Context, req *rls.TestReleaseRequest) (<-chan
 	return ch, errc
 }
 
-// Executes tiller.Ping RPC.
-func (h *Client) ping(ctx context.Context) error {
+// Executes tiller.PingTiller RPC.
+func (h *Client) pingTiller(ctx context.Context) error {
 	c, err := h.connect(ctx)
 	if err != nil {
 		return err
 	}
 	defer c.Close()
 
 	rlc := rls.NewReleaseServiceClient(c)
-	return rlc.PingTiller(ctx)
+	_, err = rlc.PingTiller(ctx, &rls.PingTillerRequest{})
```

Comment:
Hey @TheOriginalAlex, would you mind taking a look at this? Was there a way in proto code that we could actually send a request over gRPC without the need to send a request body? That would retain backwards compatibility, but I'm not too concerned given that this is a relatively new feature to Helm.

Thanks!

Reply:
I was just sending a "Ping" string earlier, sending this empty struct does the same job, I think.
https://github.com/kubernetes/helm/blob/master/pkg/proto/hapi/services/tiller.pb.go#L1084

And the server currently just returns "Pong" no matter what the input is, so I don't think this breaks backwards compatibility.
https://github.com/kubernetes/helm/blob/master/pkg/proto/hapi/services/tiller.pb.go#L1314